helm/defectdojo/templates/django-ingress.yaml

```diff
@@ -30,5 +30,5 @@ spec:
         - path: /
           backend:
             serviceName: {{ $fullName }}-django
-            servicePort: http
```

Comment:
http worked for me; it's some kind of alias I think; is this change really necessary?

Reply:
I had error, with this setup, I was using helm 3, not sure is that source of my issue, I will test it again

Reply:
I checked documentation, alias will fly for deployment and service, but it looks like for ingress will not (at least for nginx controller)